schema/validators.go

```diff
@@ -12,6 +15,14 @@ func FindEmptyColumns(table *Table, records []arrow.Record) []string {
 		for colIndex, arr := range resource.Columns() {
 			for i := 0; i < arr.Len(); i++ {
 				if arr.IsValid(i) {
+					if arrow.TypeEqual(arr.DataType(), types.ExtensionTypes.JSON) {
+						// JSON column shouldn't be empty
+						val := arr.GetOneForMarshal(i).(json.RawMessage)
+						if len(val) == 0 || string(val) == "null" || string(val) == "{}" || string(val) == "[]" {
```

Comment:
impl a separate `isEmptyJSON`:
```go
func isEmptyJSON(msg json.RawMessage) bool {
	if len(msg) == 0 {
		return true
	}
	switch string(msg) {
	case "null", "{}", "[]":
		return true
	default:
		return false
	}
}
```

Reply:
sure why not https://github.com/cloudquery/plugin-sdk/pull/1418/commits/f4c7cb6c4bdd0d4a42e18cd8f4907d4725ab85d7